cjsmin/src/chess.ts

```diff
@@ -1737,7 +1737,7 @@ export class Chess {
         };
         yield {
           move: prettyMove,
-          board: this._board,
+          board: [...this._board],
```

Comment:
If we care about performance we should find another way to do this - for now we can leave a TODO